Mathlib/Tactic/Reassoc.lean

```diff
@@ -72,6 +77,8 @@ initialize registerBuiltinAttribute {
     | _ => throwError "Constant {src} is not a theorem or definition."
     if isProtected (← getEnv) src then
       setEnv $ addProtected (← getEnv) tgt
-    ToAdditive.copyAttributes src tgt }
+    let stx := match stx? with | some stx => stx | none => #[]
+    Term.TermElabM.run' <| ToAdditive.applyAttributes stx `reasoc src tgt
```

Comment:
```suggestion
    Term.TermElabM.run' <| ToAdditive.applyAttributes stx `reassoc src tgt
```
Also, I can't remember the details now but I think attributes need to be named the same as their first atom, so `reassoc` instead of `reassocAttr`. (You can put it in an additional namespace to avoid a clash) This is because some of lean's internals use the attribute name and others use the declaration name sans namespaces or atom name.

Reply:
ok, I renamed the other occurrence of `reassoc` to avoid the name clash.